docs/backup-restore.md

```diff
@@ -0,0 +1,62 @@
+
+To easily achieve this, Scalar DL exposes pause API to make Scalar DL drain outstanding transactions and stop accepting new transactions.
+We also provide a simple client program called scalar-admin to make a pause request (and unpause request) to a Scalar DL cluster and obtain a paused duration.
+
+Note that when you use a point-in-time-restore/recovery mechanism, it is recommended to minimize the clock drifts between nodes (Scalar DL nodes and a client node that requests a pause) by using clock synchronization such as NTP.
+Otherwise, the time you get as a paused duration might be too different from the time in which the pause was actually conducted, which could restore to a point where ongoing transactions exist.
+Also, it is recommended to pause a long enough time (e.g., 10 seconds) and use the mid-time of the paused duration since clock synchronization cannot perfectly synchronize clocks between nodes.
+
+#### Database-specific ways to create a transactionally-consistent backup
+
+**Cassandra**
+
+Cassandra has a built-in replication mechanism, so you do not always have to create a transactionally-consistent backup.
+
+For example, if replication is properly set to 3 and only the data of one of the nodes in a cluster is lost, you do not need a transactionally-consistent backup because the node can be recovered with a normal (transactionally-inconsistent) snapshot and the repair mechanism. 
+However, if the quorum of nodes of a cluster loses their data, we need a transactionally-consistent backup to restore the cluster to a certain transactionally-consistent point.
+
+If you want to create a transactionally-consistent cluster-wide backup, please follow [the basic strategy](#basic-strategy-to-create-a-transactionally-consistent-backup) section, or 
+stop the Cassandra cluster and take the copies of all the nodes of the cluster, and start the cluster. 
+
+To avoid mistakes, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+Cassy is also integrated with `scalar-admin` so it can issue a pause request to the application of a Cassandra cluster.
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+**Cosmos DB**
+
+You must create a Cosmos DB account with a Continuous backup policy enabled to use point-in-time restore (PITR) feature. Backups are created continuously after it is enabled.
+To specify a transactionally-consistent restore point, please pause the Scalar DL service described in the [basic strategy](#basic-strategy-to-create-a-transactionally-consistent-backup).
+
+**DynamoDB**
+
+You must enable the point-in-time recovery (PITR) feature for DynamoDB tables. If you use Scalar DL Schema Loader, it enables PITR by default.
+To specify a transactionally-consistent restore point, please pause the Scalar DL service as described in the [basic strategy](#basic-strategy-to-create-a-transactionally-consistent-backup).
```

Comment:
```suggestion
To specify a transactionally-consistent restore point, please pause Scalar DL service as described in the [basic strategy](#basic-strategy-to-create-a-transactionally-consistent-backup).
```

Reply:
Thank you for the suggestion.
Updated document with suggested change.